src/Microsoft.Health.Dicom.SqlServer/Features/Schema/SchemaVersionConstants.cs

```diff
@@ -7,7 +7,7 @@ namespace Microsoft.Health.Dicom.SqlServer.Features.Schema
 {
     public static class SchemaVersionConstants
     {
-        public const int Min = (int)SchemaVersion.V1;
+        public const int Min = (int)SchemaVersion.V2;
```

Comment:
Do we need to roll versions when we do this? 

This change implies that anyone that wants to take this upgrade they have to migrate from schema 1 if that's where they still are.

Reply:
I can bump minor version